client/src/modules/auth/auth.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  auth,
+  AuthProvider,
+  FacebookAuthProvider,
+  GoogleAuthProvider,
+  User,
+  UserCredential,
+} from "../../firebase";
+import { login } from "./client";
+
+export const loginEmailPass = async (email: string, password: string) => {
```

Comment:
Should return user profile?

Reply:
The same for `loginWithGoogle`, `loginWithFacebook`.

Maybe also `registerEmailPass` but I'm not sure right now.